api/controller/select_books.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+import flask
+
+from api.controller.circulation_manager import CirculationManagerController
+from core.feed.acquisition import OPDSAcquisitionFeed
+from core.util.problem_detail import ProblemDetail
+
+
+class SelectBooksController(CirculationManagerController):
+    def fetch_books(self):
+        """
+        Generate an OPDS feed response containing the selected books for the
+        authenticated patron.
+
+        This method creates an OPDS acquisition feed with the books currently
+        selected by the patron and returns it as a response.
+
+        :return: A Response object.
+        """
+        patron = flask.request.patron
+
+        feed = OPDSAcquisitionFeed.selected_books_for(self.circulation, patron)
+
+        response = feed.as_response(
+            max_age=0,
+            private=True,
+            mime_types=flask.request.accept_mimetypes,
+        )
+
+        # For loans, the patron's last loan activity sync time was set. Not yet
```

Comment:
Do we see any need for this in the future for selected books? No, I think?

Reply:
Can think of any use atm